src/test/util/setup_common.cpp

```diff
@@ -222,9 +222,11 @@ ChainTestingSetup::ChainTestingSetup(const std::string& chainName, const std::ve
     m_node.sporkman = ::sporkManager.get();
     ::governance = std::make_unique<CGovernanceManager>();
     m_node.govman = ::governance.get();
+    ::triggerman = std::make_unique<CGovernanceTriggerManager>();
+    m_node.gov_trigman = ::triggerman.get();
```

Comment:
why not let `CGovernanceManager` do not let to know about `triggerman` in constructor?

In this case you can get rid of global `::triggerman`